libraries/disp/viewers/helpers/scalecontrol.h

```diff
@@ -0,0 +1,271 @@
+     * QWidget. The min and max values for the control will also be initialized.
+     *
+     * @param [in] label    text for the Qlabel
+     * @param [in] parent   parent of widget
+     * @param [in] min      min value of the control.
+     * @param [in] max      max value of the control.
+     */
+     ScaleControl(const char* label, QWidget* parent, double min, double max);
+
+    //=========================================================================================================
+     /**
+     * getUI Retunr a pointer to the GUI of the ScaleControl.
+     * @return Pointer to the ScaleControlWidget.
+     */
+    Ui::ScaleControlWidget* getUI();
+
+    //=========================================================================================================
+    /**
+     * Return the value of the control
+     * @return value of the control.
+     */
+    double value() const;
+
+    //=========================================================================================================
+    /**
+     * Set the value of the maximum sensitivity point for the non-linear sensitivity curve of the sensor.
+     * @param [in] s Maximum sensitivity value
+     */
+    void setMaxSensitivityPoint(double s);
+
+    //=========================================================================================================
+    /**
+     * Set the sensitivity value of the non-linear mapping between the control slider and the control value.
+     * @param [in] s Sensitivity, between [0, 1]
+     */
+    void setSensitivity(double s);
+
+    //=========================================================================================================
+    /**
+     * @brief setRange
+     * @param min
+     * @param max
+     */
+    void setRange(double min, double max);
+
+    //=========================================================================================================
+    /**
+     * Set the number of decimals to show in the spinbox of the control.
+     * @param d
+     */
+    void setDecimals(int d);
+
+    //=========================================================================================================
+    /**
+     * Invert the effect of sliding the control slider to the right. Slider right decreases the control value.
+     * @param [in] inverted [True = Invert the slider effect].
+     */
+    void invertSlider(bool inverted);
+
+public slots:
+
+    //=========================================================================================================
+    /**
+     * Set the value of the control.
+     * @param [in] dScale value to set the control to.
+     */
+    void setValue(double dScale);
+
+signals:
+    //=========================================================================================================
+    /**
+     * Signal emited when the value of the control changes.
+     * @param dScale
+     */
+    void valueChanged(double dScale);
+
+private:

```

Comment:
add line between param doc and return doc in all the private functions.
some functions missing parameter documentation